css/index.css

```diff
@@ -136,11 +206,139 @@ footer nav {
     display: flex;
     justify-content: space-between;
     align-items: center;
-    padding: 20px 2%;
-    font-size: 14px;
+    padding: 2% 2%;
+    font-size: 1.4rem;
 }
 
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+@media (max-width: 768px) {
+    header {
+        flex-direction: column;
+        align-items: center;
+    }
+    header img {
+        display: block;
+    }
+    header nav a {
+        width: 100%;
+        justify-content: space-around;
+        padding: 4%;
+        font-size: 1.5rem;
+    }
+
+    .top-content .text-container h2 {
+        margin-bottom: 7%;
+        margin-top: 3%;
+    }
+    .middle-content {
+        margin: 3% 2%;
+    }
+
+    .middle-content .boxes .box1 {
+        background: teal;
+    }
+    .middle-content .boxes .box3 {
+        background: gold;
+    }
+    .middle-content .boxes .box5 {
+        background: cadetblue;
+    }
+    .middle-content .boxes .box7 {
+        background: coral;
+    }
+    .middle-content .boxes .box9 {
+        background: crimson;
+    }
+
+    .bottom-content .text-container h2 {
+        margin-bottom: 7%;
+        margin-top: 3%;
+    }
+
+}
+
+@media (max-width: 400px)  {
+
+    header {
+        flex-wrap: wrap;
+        justify-content: center;
+    }
+
+    header img {
+        margin: 5% 0;
+    }
+    header nav {
+        width: 100%;
+    }
+
+    header nav {
+        width: 100%;
+        display: flex;
+        flex-direction: column;
+        margin: 0;
+        align-items: center;
+    }
+
+    header nav a {
+        width: 100%;
+        margin: 0;
+        display: block;
+        text-align: center;
+        padding: 5% 0;
+        font-size: 2rem;
+        border-bottom: 1px solid gray;
+        font-size: 1rem;
```

Comment:
Your nav font is a little small but an easy fix with the font-size. Also, font-size is written twice here so pick your mind and choose one! haha